### PR TITLE
RHDEVDOCS-4933 - Logging 5.4.13 Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.9 - 4.11

Issue: [RHDEVDOCS-4933](https://issues.redhat.com//browse/RHDEVDOCS-4933)

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cluster-logging-rn-5.4.13.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-13_{context}"]
+= Logging 5.4.13
+This release includes link:https://access.redhat.com/errata/RHBA-2023:1312[OpenShift Logging Bug Fix Release 5.4.13].
```
Comment (shipsing) [inline modules/cluster-logging-rn-5.4.13.adoc:5]: The link is not working, showing a 404 page not found error.

**Review by shipsing — APPROVED** [review]
@libander LGTM. Just a minor comment pointing to a non-working link.

**Review by periklis — COMMENTED** [review]
LGTM

File: modules/cluster-logging-rn-5.4.13.adoc (hunk 1 above)
Comment (libander) [reply modules/cluster-logging-rn-5.4.13.adoc:5]: Errata links don't work before the release goes live, this is as expected. :)

**Pull request merged**